producer-kafka/README.adoc

```diff
@@ -0,0 +1,407 @@
+            "fields":[
+               {
+                  "name":"pk_1",
+                  "type":[
+                     "null",
+                     {
+                        "type":"record",
+                        "name":"pk_1",
+                        "namespace":"",
+                        "fields":[
+                           {
+                              "name":"value",
+                              "type":"string"
+                           }
+                        ]
+                     }
+                  ],
+                  "default":null
+               },
+               {
+                  "name":"ck_1",
+                  "type":[
+                     "null",
+                     {
+                        "type":"record",
+                        "name":"ck_1",
+                        "namespace":"",
+                        "fields":[
+                           {
+                              "name":"value",
+                              "type":"int"
+                           }
+                        ]
+                     }
+                  ],
+                  "default":null
+               },
+               {
+                  "name":"col_1",
+                  "type":[
+                     "null",
+                     {
+                        "type":"record",
+                        "name":"col_1",
+                        "namespace":"",
+                        "fields":[
+                           {
+                              "name":"value",
+                              "type":[
+                                 "null",
+                                 "string"
+                              ],
+                              "default":null
+                           }
+                        ]
+                     }
+                  ],
+                  "default":null
+               }
+            ]
+         }
+      }
+   ]
+}
+----
+The name of this schema is `Value` and it is created in the same namespace as `key` schema.
+There are two required fields. The first one is `op` denoting the type of operation. It can be updated, meaning that some value in the CDC enabled table changed.
+For the update, the `op` will have a value equal to `u`. The second possible state is deleted which means that there was a delete on the field in the database.
+The deleted value is `d`. The `ts_ms` contains the timestamp of a change.
+Every field (Primary Key, Clustering Key, and normal cell) has a union schema, meaning the whole field can be absent.
+The field's actual value is also a union of the type of a field and `null`. When the value is `null`, it means that the field was cleared.
+So, to sum up, such a structure allow the CDC connector to model 3 states:
+
+==== Absence of the Value
+The absence of change is presented as a null union value. For example, if `col_1` was not modified then the Avro record will look like this:
+
+[source,json]
+----
+{
+   "op":"d",
```

Comment:
`op` `d` or `u`?... This looks like an update event to me :thinking:

Reply:
right, changed